.changeset/wild-keys-obey.md

```diff
@@ -0,0 +1,9 @@
+---
+"@salt-ds/lab": patch
+---
+
+Drawer
+
+- Replace `Scrim` with `FloatingFocusManager`
+- Remove `disableScrim`, `disableAnimations` and `scrimProps` props
+- Rename prop `isOpen` to `open` and add `onOpenChange` prop to allow component to be dismissed when clicking outside or pressing `Esc`
```

Comment:
clicking outside or Esc dont work for me in the stories :(

Reply:
oh no:( let me take a look

Reply:
Should be working now. I've moved most of the floating-ui logic into a separate hook called `useDrawer`